contracts/contracts/utils/InitializableAbstractStrategy.sol

```diff
@@ -273,6 +273,10 @@ abstract contract InitializableAbstractStrategy is Initializable, Governable {
         public
         onlyGovernor
     {
+        require(
+            !InitializableAbstractStrategy(address(this)).supportsAsset(_asset),
```

Comment:
I don't think you need `InitializableAbstractStrategy(address(this)).`. Some strategies override `supportsAsset` so you can just call `supportsAsset` directly.

Reply:
Tried that but the compiler threw an error. I'm assuming it's because this is an abstract contract but will check again

Reply:
A change from `external` to `public` was required. fixed in this commit: https://github.com/OriginProtocol/origin-dollar/pull/1869/commits/64f678a4718635e56feec3b8d3a9abcc6234f406